static/app/views/alerts/rules/issue/details/sidebar.tsx

```diff
@@ -9,18 +9,26 @@ import TimeSince from 'sentry/components/timeSince';
 import {IconChevron} from 'sentry/icons';
 import {t, tct} from 'sentry/locale';
 import {space} from 'sentry/styles/space';
-import {Actor, Member, Team} from 'sentry/types';
+import type {Actor, Member, Team} from 'sentry/types';
 import {IssueAlertRule} from 'sentry/types/alerts';
+import {useApiQuery} from 'sentry/utils/queryClient';
+import useOrganization from 'sentry/utils/useOrganization';
 
 import {TextAction, TextCondition} from './textRule';
 
 type Props = {
-  memberList: Member[];
+  projectSlug: string;
   rule: IssueAlertRule;
   teams: Team[];
 };
 
-function Conditions({rule, teams, memberList}: Props) {
+function Conditions({rule, teams, projectSlug}: Props) {
+  const organization = useOrganization();
+  const {data: memberList} = useApiQuery<Member[]>(
```

Comment:
Not necessary to do now of course but it seems like we can/should replace other member list calls with this in the future